docs/UserManuals/ConfigUI/GitHub.md

```diff
@@ -120,6 +120,13 @@ If you're using GitHub Action to conduct `deployments`, please select "Detect De
 - Deployment: A GitHub Action job with a name that matches the given regEx will be considered as a deployment.
 - Production: A GitHub Action job with a name that matches the given regEx will be considered a job in the production environment.
 
+A GitHub workflow run has many jobs. Each GitHub workflow run is converted to a 
+cicd_pipeline in the domain layer and each GitHub Action job is converted to a cicd_task in the domain layer.
```

Comment:
"each GitHub Action job" -> "each GitHub Actions job run"

Reply:
Hmm, but according to github definition, it's just a job.. not a job run